kolibri/plugins/coach/assets/src/views/exams-page/index.vue

```diff
@@ -118,11 +118,11 @@
     name: 'coachExamsPage',
     $trs: {
       exams: 'Exams',
-      classExams: '{className} Exams',
+      classExams: '{className} exams',
```

Comment:
while we're here, maybe we can change this to:

```
classExams: "Exams in '{className}'",
```

Reply:
Makes total sense, will change asap!